test/e2e/data/tiles/wgs84/0/r0_c0.mapml

```diff
@@ -2176,15 +2176,15 @@
                   <tr><th scope="row">subunit</th><td itemprop="subunit">Ivory Coast</td></tr>
                   <tr><th scope="row">su_a3</th><td itemprop="su_a3">CIV</td></tr>
                   <tr><th scope="row">brk_diff</th><td itemprop="brk_diff">0</td></tr>
-                  <tr><th scope="row">name</th><td itemprop="name">C�te d'Ivoire</td></tr>
-                  <tr><th scope="row">name_long</th><td itemprop="name_long">C�te d'Ivoire</td></tr>
+                  <tr><th scope="row">name</th><td itemprop="name">C�te d'Ivoire</td></tr>
+                  <tr><th scope="row">name_long</th><td itemprop="name_long">C�te d'Ivoire</td></tr>
```

Comment:
Many "modifications" like these can be seen in this PR, I in fact did not modify these lines (and nothing actually changed AFAICT). This is certainly due to the "�" characters. Should be fixed, separately from this PR.

Reply:
That was careless of me.